updated your guidelines for core node "inject" (#287)
node-red/node-red.github.io: docs/user-guide/nodes.md

 - a String, number, boolean, Buffer or Object
 - a Timestamp in milliseconds since January 1st, 1970

> The `interval` can maximum be set to every 596 hours (equivalent of about 24 days).

Reviewer: ```suggestion
The `interval` can be set up to a maximum of 596 hours (approximately 24 days).
```

If you are looking at intervals greater than one day - consider using a scheduler node that can cope with power outages and restarts.

> The `interval between tmes` and `at a specific time` options use the standard cron system. This means that 20 minutes will be at the next hour, 20 minutes past and 40 minutes past - not in 20 minutes time. If you want every 20 minutes from now - use the `interval` option.

Reviewer: ```suggestion
The `interval between times` and `at a specific time` options use the standard cron system. This means that 20 minutes will be at the next hour, 20 minutes past and 40 minutes past - not in 20 minutes time. If you want every 20 minutes from now - use the `interval` option.
```

typo 'tmes'
